apps/tmbitwarden.sh

```diff
@@ -18,6 +17,16 @@ debug_mode
 # Check if root
 root_check
 
+# Check if Bitwarden is already installed
+print_text_in_color "$ICyan" "Checking if Bitwarden is already installed..."
+if [ -d "root/bwdata/docker" ]
+then
+    msg_box "It seems like 'Bitwarden' is already installed.\nYou cannot run this script twice, because you would loose all your passwords."
+    exit 1
+fi
+
+print_text_in_color "$ICyan" "Installing Bitwarden password manager..."
+
 msg_box "Bitwarden is a password manager that is seperate from Nextcloud, though we provide this service because it's self hosted and secure.
 
 If you just want to run Bitwarden locally (not connecting your smartphone) then you can use 'localhost' as domain.
```

Comment:
:+1:

Reply:
But, use the checks from the update script to really check if Bitwarden is installed or not: https://github.com/nextcloud/vm/blob/master/nextcloud_update.sh#L557-L577

Reply:
Okay

Reply:
this should be done.